src/EmbeddedExplorer.ts

```diff
@@ -141,7 +123,7 @@ export class EmbeddedExplorer {
         ? encodeURIComponent(JSON.stringify(headers))
         : undefined,
       shouldPersistState: !!persistExplorerState,
-      sendRequestsFrom: sendRequestsFrom ?? 'parent',
+      sendRequestsFrom: 'parent',
```

Comment:
should we remove the option if it can only have one value?

Reply:
This is a required arg, so we would be removing the dependency it in studio ui, but some folks might be on an old version of the cdn (pinned version?) that expected that query param, so I was trying to avoid that. Do you think its cool to just kill it on the studio ui side and always say sendRequestsFrom parent? Dev tools sends from parent as well.

Reply:
I don't actually think anyone is on a pinned version of the embed now that I think of it

Reply:
what would happen for people on an old version of the cdn if we removed this field from the options? (would it not be a no-op for them?)

Reply:
We would ignore their choice in the config & always send from parent. Things would break if they had passed `sendRequestFrom=embed`.

Can you remember anywhere we needed `sendRequestFrom=embed`?

Reply:
@cheapsteak [we don't have it documented in the studio docs either ](https://www.apollographql.com/docs/studio/explorer/embed-explorer/)

Reply:
I don't recall us having needed `sendRequestFrom=embed` ourselves

Since we're removing the ability to send the request from the studio-ui side anyway, I don't believe it makes a functional difference if we expose a type of `sendRequestsFrom: 'parent'` on this field? Things would break if they had passed `sendRequestFrom=embed` anyway

Someone consuming this via NPM may get a typescript error if they use typescript once they upgrade, but that's fine. It won't crash their running app. We can bump the minor version since we're still in the `0.x.x`s

Reply:
I am confused - I thought we were discussing if we want to remove it from studio-ui? I would need to open a PR to remove it there before removing it here. 

Going to merge as is and leave this ticket open & follow up by removing on the studio-ui side if that is what we want to do!

Reply:
oh yes we do want to disallow embeds from sending requests from studio-ui 

> We want to prevent a scenario where a 3rd party site could load an invisible embedded explorer with an arbitrary query and have that query be submitted from a page on our domain

the goal is to make this impossible, not just impossible for newer versions

Reply:
PR up here! https://github.com/mdg-private/studio-ui/pull/6154

What confuses me is the actual change is on the studio side, not in this repo. It doesn't really represent a minor change because you can't reset to an older version and get that param back.

Reply:
Also @cheapsteak I realized that if folks don't have updated studio bundles, merging a PR where this param is deleted will break embed for those folks. I will wait until https://github.com/mdg-private/studio-ui/pull/6154 is merged for a week or so and then merge this one after https://github.com/apollographql/embeddable-explorer/pull/64

Reply:
🙏 

indeed, we hopefully wouldn't need to make more of these, special situation since it relates to security  

a reason to err towards limiting surface area for public APIs, especially ones that touch the bridge between the embed and studio

Reply:
> if folks don't have updated studio bundles, merging a PR where this param is deleted will break embed for those folks.


ahh it's a boolean that defaults to a different value on the studio-ui end 
👍 yup sounds good

Reply:
thanks for keeping track of backwards compatibility while making these changes 🙏 🙏